media-libs/libjxl/libjxl-0.9.1.ebuild

```
@@ -0,0 +1,91 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib flag-o-matic
+
+# This changes frequently.  Please check the testdata submodule when bumping.
+TESTDATA_COMMIT="ff8d743aaba05b3014f17e5475e576242fa979fc"
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+SRC_URI="https://github.com/libjxl/libjxl/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	test? ( https://github.com/libjxl/testdata/archive/${TESTDATA_COMMIT}.tar.gz
+		-> ${PN}-testdata-${TESTDATA_COMMIT}.tar.gz )"
```

```suggestion
SRC_URI="
	https://github.com/libjxl/libjxl/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
	test? (
		https://github.com/libjxl/testdata/archive/${TESTDATA_COMMIT}.tar.gz
			-> ${PN}-testdata-${TESTDATA_COMMIT}.tar.gz
	)
"
```

Don't use hanging parenthesis, they make additions harder and the overall code is less readable.